ChangeLog.md

```diff
@@ -2,6 +2,11 @@
 
 ## 09/19/2022 3.1.0
 
+- Retire some features
+  - Retire commands "generate-uml"
+  - Retire commands "generate-wireformat"
+  - Retire commands "resolve-spec"
+  - Retire Autorest plugin
```

Comment:
need use a new minor version number

Reply:
@keryul , need to bump minor version as 3.2.0